_posts/2019/2019-09-20-linux-party-experience.md

```diff
@@ -0,0 +1,158 @@
+
+# Preparing a bootable media
+
+Ubuntu provided a great guide of how you can prepare a bootable media [here](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows#0)
+
+Here we would like to stress on point 4, Boot selection and Partition scheme, to keep you partition scheme as MBR and this wouldn't cause a problem, though you can give the other options a try but they require more care in the installation process
+
+# Understanding EFI tables and custom partitions
+
+EFI tables- In a 'not that technical' language it is a partition(like the C: or D: drive on your windows) which stores 'bootloaders' of various OS present on your system
+Now about 'bootloaders', they are programs that fire up your OS on startup.
+On a Windows laptop, you would be having the Windows Bootloader installed by default on an EFI partition, you can check this by opening Disk Management from the Start menu.
+Now this knowledge will come in useful later on
+
+You may be used to using Windows partition scheme in which one makes different drives like C:, D:, etc but Linux uses a different Partition scheme.
+
+```
+/
+├── bin
+├── boot
+├── dev
+├── etc
+├── home
+├── lib
+├── lib64
+├── lost+found
+├── media
+├── mnt
+├── opt
+├── proc
+├── root
+├── run
+├── sbin
+├── snap
+├── srv
+├── sys
+├── tmp
+├── usr
+├── var
+```
+
+So what you see above is the list of directories that were in my '/'(pronounced 'root') directory. So instead of providing different drives for segregating data Linux gives a master directory that contains everything and within this, several directories can be allocated separate space on your hard disk so it would be somewhat like a different drive. If you don't give allocate separate space to separate directories then would be sharing root's allocated space by default.
+
+[This](https://www.youtube.com/watch?v=HbgzrKJvDRw) is a great video explaining the Linux filesystem in detail and don't worry if a lot of things go over your head, for starters, we are only concerned with /, /home and swap
+
+/home is where a user's files are stored(similar to the C:\Users in windows) but you would have to allocate more space than what you might be using in C:\Users because ALL of the user files will be in /home and there would be no other D: or E: drive to store these files
+
+swap is a 'virtual RAM' that is utilised to extend RAM beyond its physical capacity, with swap memory an 8 GB RAM can 'store' much more data than 8 GB though it would be much slower.
+
+Now for all this blabbering from our experience a general space ratio of 40:60 between / and /home and having swap as double the size of your RAM is the ideal composition. Also, root requires a minimum of 20 GB for smooth operation
+
+## The actual partitioning
+
+Now we have three cases
+
+* Only SSD or only HDD
+* SSD + HDD
+* Optane + HDD
+
+Now for only HDD laptops, the work is pretty straightforward that you have to shrink the required memory which you have calculated previously. 
```

Comment:
keep only after HDD or SSD, whatever the case may be. Fix this for all occurences